src/components/ImageBanner/ImageBanner_spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+import { shallow, mount } from 'enzyme'
+
+import ImageBanner, { BackgroundImage } from './ImageBanner'
+import BannerTitle from '../BannerTitle/BannerTitle'
+import BannerSubtitle from '../BannerSubtitle/BannerSubtitle'
+
+describe('ImageBanner', () => {
+  it('should render', () => {
+    const component = renderer.create(<ImageBanner />).toJSON()
+    expect(component).toMatchSnapshot()
+  })
+
+  it('should render a <BannerTitle />', () => {
+    const wrapper = shallow(<ImageBanner />)
+    expect(wrapper.find(BannerTitle).length).toBe(1)
+  })
+
+  it('should render a <BannerSubtitle />', () => {
+    const wrapper = shallow(<ImageBanner />)
+    expect(wrapper.find(BannerSubtitle).length).toBe(1)
+  })
+
+  it('should render the titleText from props to BannerTitle ', () => {
+    const titleText = 'Here is a test title!'
+    const wrapper = mount(<ImageBanner />)
+    wrapper.setProps({ titleText })
+    expect(wrapper.find(BannerTitle).text()).toBe(titleText)
+  })
+
+  it('should render the subtitleText from props to BannerTitle ', () => {
+    const subtitleText = 'And here is a test subtitle!'
+    const wrapper = mount(<ImageBanner />)
```

Comment:
Do we have to mount these? Mounting is slow etc

Reply:
added note: if you need to test for any child components that don't get rendered with shallow(), then use dive() to render the next layer of components as opposed to mount() which renders the whole thing

Reply:
I had a go at this using `dive()`. The issue is when it finds the component the text it contains is  styled component html and the test fails. Will post an example snippet when I have a bit more time, I wanted the rest of the changes to be updated though just so that things aren't blocked.